.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,81 @@
+---
+name: "\U0001F41B Bug Report"
+about: "Please report issues related to the TypeScript and HTML API of Vaadin components here."
+title: ''
+labels: 'i: needs triage'
+assignees: ''
+
+---
+
+<!--
+  Please fill out each section below, otherwise, your issue risks taking an infinite amount of time to triage. This info allows the maintainers team to diagnose and fix your issue as quickly as possible.
+
+  Before opening a new issue, please search existing issues: https://github.com/vaadin/web-components/issues.
+-->
+
+### Description
+
+Describe the issue that you're seeing.
+
+<!-- Example: The `vaadin-grid` element scrolls when applying filter. -->
+
+### Expected outcome
+
+What should happen?
+
+<!-- Example: No scrolling should happen. -->
+
+### Actual outcome
+
+What happened.
+
+<!-- Example: Grid content occasionally gets scrolled. -->
+
+### Live Demo
+
+A screenshot or video with issue reproduced.
```

Comment:
This could be marked as optional IMO. I'd rather that the reporter invests time into a reproduction.